packages/orbit-components/src/primitives/MobileDialogPrimitive/components/DialogContent.tsx

```diff
@@ -140,21 +48,40 @@ const DialogContent = ({
     [onClose],
   );
   return (
-    <StyledDialog role="dialog" id={dialogId} data-test={dataTest}>
-      <StyledDialogOverlay shown={shown} ref={overlay} onClick={handleClickOutside} />
-      <StyledDialogWrapper
-        shown={shown}
+    <div className="w-full" role="dialog" id={dialogId} data-test={dataTest}>
+      {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
+      <div
+        className={cx(
+          "fixed inset-0 z-[10011] h-full w-full bg-[rgba(23,27,30,0.6)]",
```

Comment:
Non-blocking. Just for discussion. I do not think this z-index value is okay to use, that's too big and should probably just be bigger than the rest of our values or `zIndexOnTheTop` 🤔

Reply:
those were like this before. we can clean up z-indices later on

Reply:
alright 👍🏻